modin/pandas/dataframe.py

```diff
@@ -64,6 +64,9 @@
     cast_function_modin2pandas,
 )
 
+# Dictionary of extensions assigned to this class
+_DATAFRAME_EXTENSIONS_ = {}
```

Comment:
Should we add a test for each object extension (df, series, pd)?

Reply:
👍 Yes, I haven't added tests yet

Reply:
I pushed some tests, where in CI do you think would be best to add @YarShev or @anmyachev?

Reply:
Added tests there, they are very short.